### PR TITLE
[CFM-817] Adds a settings page to configure the disclaimer settings.

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-817

Screenshot:
disclaimer

### CODE REVIEW

**Comment by soniCaH** [comment]
@DavidHernandez please move this code to a more appropriate module.
{{c4m_ds}} is holding different Display Suite fields/templates/...
See {{c4m_admin}} for example.

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/c4m_ds.pages.inc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ */
+
+/**
+ * Form definition for admins to manage the disclaimer text.
+ */
+function c4m_ds_manage_disclaimer_form($form, $form_state) {
+  $form['c4m_ds_content_disclaimer'] = array(
+    '#title' => t('Disclaimer text'),
+    '#type' => 'textarea',
+    '#default_value' => variable_get('c4m_ds_content_disclaimer', 'DISCLAIMER: This information is provided in the interests of knowledge sharing and capacity development and should not be interpreted as the official view of the European Commission, or any other organisation.'),
```
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/c4m_ds.pages.inc]: Please update the variable name as well. Has nothing to do with display suite, so don't know why it was every called like that.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/c4m_ds.pages.inc]: @soniCaH 
On the ticket it is written explicitly to use this variable name
selection_273

**Comment by soniCaH** [comment]
Yes, that's why I added the sentence "don't know why it was ever called like that" :-).
Moved it.

**Pull request merged**